app/javascript/components/patient/PatientPage.js

```diff
@@ -20,7 +20,7 @@ class PatientPage extends React.Component {
             details={{ ...this.props.patient, blocked_sms: this.props.blocked_sms }}
             hoh={this.props.other_household_members.find(patient => patient.head_of_household)}
             collapse={this.props.can_modify_subject_status}
-            edit_mode={false}
+            hidePreviousButton={false}
```

Comment:
same comment here - leave `edit_mode`

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/6620f09e6346a83c99080db3e9e12eef8a1ee5f4